Update copy and link for Outfunnel app (#5386)
PostHog/posthog.com: contents/docs/apps/outfunnel-exporter.md

---
title: Outfunnel Exporter
github: https://github.com/PostHog/outfunnel-export-plugin
installUrl: https://app.posthog.com/project/apps?name=Outfunnel
thumbnail: ../../apps/thumbnails/outfunnel-logo.png
tags:
    - outfunnel-export
---

> ### What does ProductSync by Outfunnel do?

Reviewer: ```suggestion
### What does the Outfunnel Exporter do?
```

> ProductSync by Outfunnel sends events from PostHog to Outfunnel as they are ingested. Outfunnel is a platform for prioritizing leads based on product, marketing and sales engagements.

Reviewer: ```suggestion
The Outfunnel Exporter sends events from PostHog to Outfunnel as they are ingested. Outfunnel is a platform for prioritizing leads based on product, marketing and sales engagements.
```

### What are the requirements for this app?

> ProductSync by Outfunnel requires either PostHog Cloud, or a self-hosted PostHog instance running [version 1.30.0](https://posthog.com/blog/the-posthog-array-1-30-0) or later.

Reviewer: ```suggestion
The Outfunnel Exporter requires either PostHog Cloud, or a self-hosted PostHog instance running [version 1.30.0](https://posthog.com/blog/the-posthog-array-1-30-0) or later.
```

### Is the source code for this app available?

> PostHog is open-source and so are all apps on the platform. The [source code for the ProductSync by Outfunnel](https://github.com/PostHog/outfunnel-export-plugin) is available on GitHub.

Reviewer: ```suggestion
PostHog is open-source and so are all apps on the platform. The [source code for the Outfunnel Exporter](https://github.com/PostHog/outfunnel-export-plugin) is available on GitHub.
```
